documentation/sphinx/source/api-general.rst

```diff
@@ -104,3 +104,34 @@ If you want to set the same option multiple times (e.g. to add multiple client l
 	FDB_NETWORK_OPTION_EXTERNAL_CLIENT_DIRECTORY = /path/to/dir1:/path/to/dir2:...
 
 Network options specified using environment variables are set at the end of the call to set the API version. They will be applied in the order of the corresponding option codes assigned to the options. If there is an error reading the appropriate environment variables or if the option cannot be set with the specified value, then the call to set the API version may return an error. In that case, you may attempt to set the API version again, but you must do so with the same version as the attempt that failed.
+
+.. _client-network-thread:
+
+Client network thread
+=====================
+
+FoundationDB clients start a thread that is used to serialize operations and communicate with the cluster. This thread is commonly referred to as the network thread, and most operations performed by a client (such as reads, writes, and commits) will take place on this thread. It is important that a client application does not block on this thread, such as by issuing a blocking call in a callback from a FoundationDB operation. Some client language bindings (e.g. Java) will protect you from this risk by notifying the application that an operation has completed from a different thread automatically.
+
+.. _client-network-thread-performance:
+
+Performance
+-----------
+
+A client process will only ever have one network thread running, and as a result that process may be limited in the its throughput by the amount of work that can be performed on the network thread. If a client application needs to support higher throughput than a single network thread can provide, then more network threads can be started by running additional client processes.
```

Comment:
```suggestion
A client process can only have *one* network thread running, and as a result that process may be limited in the its throughput by the amount of work that can be performed on the network thread. If a client application needs to support higher throughput than a single network thread can provide, then more network threads can be started by running additional client processes.
```

Reply:
Maybe I'll reword this a bit. I was trying to also hint at the fact that a network thread can't be stopped and restarted, but I think I should make that more explicit.